examples/getstarted/src/admin/app.jsx

```diff
@@ -5,7 +5,7 @@ import { Button } from '@strapi/design-system';
 import { registerPreviewRoute } from './preview';
 
 const config = {
-  locales: ['it', 'es', 'en', 'en-GB'],
+  locales: ['it', 'es', 'en', 'en-GB', 'ru'],
```

Comment:
```suggestion
  locales: ['it', 'es', 'en', 'en-GB'],
```

Hi, this PR looks good thanks!

If you could please just remove the addition of the russian locale here. We don't really want to change the default config of this example 🙏🏻